exonum/src/messages2/spec.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// A low-level versions of `transactions!` macro, which generates structs for messages,
+/// but does not require the messages to implement `Transaction`.
+#[macro_export]
+macro_rules! messages {
```

Comment:
While we're here... Do we still need this magic macro with the new messages?

Reply:
We need it to keep compatible with old `transactions!` macro